content/who-we-are/team/_index.en.md

```diff
@@ -12,12 +13,27 @@ in_progress: false
 teams:
   - title: Board
     description: The board of directors is responsible for strategic leadership of DIVD.
-    members: []
+    members:
+      - /who-we-are/team/people/eleonora-petridou
```

Comment:
Geliefde deze niet mee te comitten, anders gaat het script er mee om als een override

Reply:
Dan kan deze :yeet: toch?

Reply:
Wat moet ik precies niet mee committen?

Reply:
kunnen we pre commit hooks introduceren die dit tegenhouden? :)
lefthook oid